src/helpers/grid.test.js

```diff
@@ -115,7 +119,7 @@ describe('grid system', () => {
           box-sizing: border-box;
           width: 100%;
           padding: 0 15px; }
-          @media (min-width: 46.25em) {
+          @media (min-width: 40.0625em) {
```

Comment:
What's the reason behind the change of these values?

Reply:
Good question - it used to use the default `sass-mq` breakpoints, now it's using the ones defined in our settings.

Reply:
Could we add this to the PR comments?

Reply:
Done.